patches/node/darwin_libuv_use_posix_spawn.patch

```diff
@@ -0,0 +1,574 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Pat DeSantis <pdesantis3@gmail.com>
+Date: Mon, 25 Jan 2021 15:53:44 -0500
+Subject: darwin,libuv: use posix_spawn
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Spawning child processes in an Electron application with a hardened runtime has become slow in macOS Big Sur.
+
+This patch is a squashed version of https://github.com/libuv/libuv/pull/3064, with the addition of API availability annotations to fix a build warning (since Electron compiles with the `-Wunguarded-availability-new` flag). This patch should be removed when libuv PR 3064 is merged.
```

Comment:
> with the addition of API availability annotations to fix a build warning (since Electron compiles with the `-Wunguarded-availability-new` flag)

Looking at the patch, this comment is no longer valid. Right ?

Reply:
Good catch, I'll update the comment!